companies/baaderbank.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "baaderbank",
+    "relevant-countries": [
+        "de",
+        "at"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Baader Bank AG",
+    "address": "Weihenstephaner Straße 4\n85716 Unterschleißheim\nDeutschland",
+    "phone": "+49 89 5150 0",
+    "fax": "+49 89 5150 1111",
+    "email": "datenschutz@baaderbank.de",
```

Comment:
As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.